### PR TITLE
docs: move projects from README.md to docs/userguides/projects.md

### PR DESCRIPTION
**Pull request by sabotagebeats**
### What I did

updated README.md to move projects to projects.md

fixes: APE-403

### How I did it

followed the ticket as best I could

### How to verify it

review documentation

### Checklist

- [x] All changes are completed
- [ ] New test cases have been added
- [x] Documentation has been updated

### CODE REVIEW

**Comment by sabotagebeats** [comment]
ape-init is already covered in projects guide.

**Review by unparalleled-js — CHANGES_REQUESTED** [review]
The readme has headers, a short description, and a link to the more official guide for each of the sub-parts. Like if you look below there is one for `accounts`... Why do you specifically to remove only the `project` one?

What is the goal with this one?

Edit: I am guess the main intent is to just remove the `ape init` part, so let's put back the header?

File: README.md [hunk 1]
```diff
@@ -84,15 +84,7 @@ Install 3rd party plugins at your own risk.
 
 Additionally, plugins that come bundled with **`ape`** in the core installation cannot be removed and are part of the **`ape`** core software.
 
-## Projects
-
-When using Ape, you generally will work with a project.
-
-```bash
-ape init
```
Comment (unparalleled-js) [inline README.md:92]: Maybe just remove the `ape init` part

File: README.md [hunk 2]
```diff
@@ -84,15 +84,7 @@ Install 3rd party plugins at your own risk.
 
 Additionally, plugins that come bundled with **`ape`** in the core installation cannot be removed and are part of the **`ape`** core software.
 
-## Projects
```
Comment (unparalleled-js) [inline README.md:87]: Keep the header

File: README.md [hunk 3]
```diff
@@ -84,15 +84,7 @@ Install 3rd party plugins at your own risk.
...
-
-When using Ape, you generally will work with a project.
```
Comment (unparalleled-js) [inline README.md:89]: and maybe keep this sentence too or at least have 1 sentence explaining what we mean by `project`.

**Comment by sabotagebeats** [comment]
In addition to above comments, may need reordering to move projects section of README.md out of plugins section of README.md

**Pull request merged**